Artifacts/windows-python3/artifactfile.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2015-01-01/dtlArtifacts.json",
+  "title": "Python 3",
+  "description": "Installs Python 3 (latest version) ",
```

Comment:
Assuming a newer version comes out, this will no longer be the latest. Probably best not to add this comment, as it will become out of date and will require a code change to update.